### PR TITLE
Update 18f-agile-based-project-approach.md with content on agilefall

### PR DESCRIPTION
**Pull request by porta-antiporta**
updating to include blurb on agilefall. @hannahkane @suprenant thoughts?

### CODE REVIEW

File: pages/18f-agile-based-project-approach.md [hunk 1]
```diff
@@ -18,4 +18,6 @@ Not all of our projects are greenfield; sometimes we work on legacy systems. Inc
...
 
+A common pitfall is treating agile as a silver bullet to all that ails software development; agile is viewed as risk free.  Thus when an agile projects fail, agile is blamed for it.  Agile does not eliminate risk comepletely, it provides techniques to manage risk more effectively than traditional waterfall processes.  Unlike waterfall, agile accepts that the unknowns will lead to change.  Agile treats change as an integral part of the process, as opposed to exceptions that need to be resolved via change control mechanisms.  This enables agile teams to manage risk by allowing change to drive course corrections.  However, some agile adoptions focus on the ceremonies as opposed to the intent. Incomplete, incorrect, or uninformed adoption of agile techniques can lead to an ineffective process we call “agilefall.” 18F has written a blog post about [how to avoid this pitfall](https://18f.gsa.gov/2015/12/29/is-your-project-using-agilefall/). Effective agile adoption will enable an organization to be nimble and respond effectively to the inevitable change that arises during software development.
```
Comment (hannahkane) [inline pages/18f-agile-based-project-approach.md]: The first two sentences confused me a little bit. Suggest rewriting to be something like: "A common pitfall is expecting agile to be a silver bullet to all that ails software development, and to expect agile to eliminate all project risks. We've seen teams try agile and, when the practices fail to eliminate all the challenges, write off agile entirely, claiming "agile doesn't work." In reality, agile does not eliminate risk completely..." [also noting a typo in the word "completely"]

Love the rest of the paragraph.

**Review by hannahkane — COMMENTED** [review]
made a suggestion to clarify some language

File: pages/18f-agile-based-project-approach.md (hunk 1 above)
Comment (porta-antiporta) [reply pages/18f-agile-based-project-approach.md]: 👍

Comment (porta-antiporta) [reply pages/18f-agile-based-project-approach.md]: made said changes and merged into `agilefall`

**Pull request merged**